model/src/main/kotlin/Package.kt

```diff
@@ -107,7 +107,13 @@ data class Package(
      * for dependency versions.
      */
     @JsonInclude(JsonInclude.Include.NON_DEFAULT)
-    val isMetaDataOnly: Boolean = false
+    val isMetaDataOnly: Boolean = false,
+
+    /**
+     * Indicates whether this [Package] has been modified compared to to its original repository.
```

Comment:
Duplicate "to to". Also, we should not assume here that the source code comes from a repository, it could also be a source artifact instead. So, maybe better: "Indicates whether the source code of this [Package] has been modified compared to the original source code, e.g. in case of a fork of an upstream Open Source project".